client/src/main/java/org/dst/client/cmd/DstCmdStarter.java

```diff
@@ -15,12 +16,30 @@
           commandHandlers = new HashMap<>();
 
   public static void main(String[] args) {
-    //TODO(jyx) Check the server is open or not
 
+    String address;
     if (args.length == 0) {
-      client = new DefaultDstClient(defaultAddress);
+      address = defaultAddress;
+    } else {
+      try {
+        address = ArgsParseUtil.covertArgsToAddress(args);
+      } catch (DstException e) {
+        System.out.println(e.getMessage());
+        return;
+      }
+    }
+
+    try {
+      client = new DefaultDstClient(address);
+    } catch (NullPointerException e) {
```

Comment:
How about use Exception instead.
npe may be too fine grain size

Reply:
I have used Exception instead.